app/controllers/reports_controller.rb

```diff
@@ -22,6 +22,7 @@ def edit
   # POST /reports or /reports.json
   def create
     @report = Report.new(report_params)
+    @report.user = current_user
```

Comment:
You can insert `current_user` into `report_params` using `Hash#merge`

```suggestion
    @report = Report.new(report_params.merge(user_id: current_user.id)
```

`Hash#merge` works like
```rb
report_params = {}
report_params.merge(user_id: 4)
# => { :user_id => 4 }
```

how does that sound?